Commands/settings.py

```diff
@@ -0,0 +1,101 @@
+import discord
+import asyncio
+
+from discord.ext import commands
+from .GuildData import get_guild_data, save_configs
+
+
+class Settings(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="add_admin", brief="**admin-only** \n\u2003 Add a bot admin", usage="<mentioned user>", help="*admin-only* \u2003 Add a bot admin. The user will be given access to the bot commands that are labelled **admin-only**. The user will not be given the role of server admin. \n\n<mention user> \u2003 Mention the user that you want to make a bot admin.")
+    async def add_admin(self, ctx, mention):
+        guild_data = await get_guild_data(ctx.message.guild.id)
+
+        if not (ctx.message.author.guild_permissions.administrator or ctx.author.id in guild_data.bot_admins):
+            # if the user is not a bot admin or server admin
+            await ctx.send("https://gph.is/g/4w8PDNj")
+            return
+        elif not ctx.message.mentions:
```

Comment:
this can be a normal if instead of an elif because you return in the previous if, right?